### PR TITLE
Add hugo docsite content

### PR DESCRIPTION
**Pull request by moficodes**
#### What type of PR is this?

/kind documentation
#### Which issue(s) this PR fixes:

Partially #377

### CODE REVIEW

**Comment by moficodes** [comment]
@alculquicondor

**Comment by moficodes** [comment]
@alculquicondor did you get a chance to review this?
Need this merged to start the domain request process.

File: site/content/en/docs/Concepts/_index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+linkTitle: "Concepts"
+weight: 4
+description: >
+  Core Kueue Concepts
+---
+
+This section of the documentation helps you learn about the components, APIs and
+abstractions that Kueue uses to represent your cluster and workloads.
+
+## Glossary
```
Comment (alculquicondor) [inline site/content/en/docs/Concepts/_index.md:12]: Also have https://github.com/kubernetes-sigs/kueue/tree/main/docs/concepts#apis here

File: site/content/en/about/_index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</div>
+
+{{< /blocks/section >}}
+
+
+
+{{< blocks/section >}}
+
+<div class="col-12">
+<h1 class="text-center">Kueue is a lean controller that you can install on top of a vanilla Kubernetes cluster. Kueue does not replace any existing Kubernetes components. </h1>
```
[the remark this answers is not in the document]
Comment (moficodes) [reply site/content/en/about/_index.html:33]: I guess we could just remove the about page? 

Have the about docs in root and link to doc from there?

File: site/config.toml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+  # A variable used in various docs to determine URLs for config files etc.
+  # To find occurrences, search the repo for 'params "githubbranch"'.
+  githubbranch = "main"
+
+  # These entries appear in the drop-down menu at the top of the website.
+  [[params.versions]]
+    version = "main"
+    githubbranch = "main"
+    url = "https://kueue.sigs.k8s.io"
```
Comment (kerthcet) [inline site/config.toml:118]: Same as @alculquicondor suggested, it's unaccessible now.

File: site/content/en/docs/Concepts/_index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
```
[the remark this answers is not in the document]
Comment (kerthcet) [reply site/content/en/docs/Concepts/_index.md:1]: I think we can move most of our docs/posts to the site and link to the site in the project repo, is this our plan? Then we can avoid the duplication.

[the remark this answers is not in the document]
Comment (alculquicondor) [reply site/content/en/docs/Concepts/_index.md:1]: Yes. There will be an transitory time when the website is not live yet, so anybody looking for the documentation would have to read it from github in the site/content folder

File: site/config.toml [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  github_repo = "https://github.com/kubernetes-sigs/kueue"
+  github_project_repo = "https://github.com/kubernetes-sigs/kueue"
+  github_subdir = "site"
+  RSSLink = "/index.xml"
+  author = "kueue" # add your company name
+  github = "kueue" # add your github profile name
+  twitter = "kueue" # add your twitter profile
+
+  copyright = "The Kueue Authors."
+  privacy_policy = "https://policies.google.com/privacy"
```
[the remark this answers is not in the document]
Comment (moficodes) [reply site/config.toml:89]: kubeflow, knative both have this. 
So I added it too.
Not sure if needed.

[the remark this answers is not in the document]
Comment (alculquicondor) [reply site/config.toml:89]: Kubeflow is owned by Google (the ownership transfer to CNCF is in the works).

Knative used to belong to Google. They might have forgotten to remove this line.

File: site/config.toml [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+  # version number for this doc set.
+  version = "main"
+
+  # Flag used in the "version-banner" partial to decide whether to display a
+  # banner on every page indicating that this is an archived version of the docs.
+  archived_version = false
+
+  # A link to latest version of the docs. Used in the "version-banner" partial to
+  # point people to the main doc site.
+  url_latest_version = "https://sigs.k8s.io/kueue/docs"
```
Comment (alculquicondor) [inline site/config.toml:108]: We are settled on removing the existing docs.

Update this to https://sigs.k8s.io/kueue/site/content/en/docs until we have a domain.

File: site/content/en/_index.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
++++
+title = "Kueue"
```
[the remark this answers is not in the document]
Comment (alculquicondor) [reply site/content/en/_index.html:2]: Can we do this?

File: site/content/en/_index.html [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+{{% blocks/lead color="primary" %}}
+<h1 class="text-center">Kueue is a set of APIs and controller for job queueing.</h1>
+{{% /blocks/lead %}}
+
+{{% blocks/lead color="secondary" %}}
+<h1 class="text-center">Kueue is a job-level manager that decides when a job should be admitted to start (as in pods can be created) and when it should stop (as in active pods should be deleted).</h1>
+{{% /blocks/lead %}}
+
+{{% blocks/lead color="dark" %}}
+<h1 class="text-center">Kueue is a lean controller that you can install on top of a vanilla Kubernetes cluster. Kueue does not replace any existing Kubernetes components. </h1>
```
Comment (alculquicondor) [inline site/content/en/_index.html:30]:
```suggestion
<h1 class="text-center">Kueue is a lean controller that you can install on top of a vanilla Kubernetes cluster, to build a multi tenant batch system. Kueue does not replace any existing Kubernetes components. </h1>
```

File: site/content/en/docs/_index.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+---
+title: "Kueue"
+linkTitle: "Documentation"
+weight: 20
+menu:
+  main:
+    weight: 20
+---
```
Comment (alculquicondor) [inline site/content/en/docs/_index.md:8]: Should we list the three main sections here like https://github.com/kubernetes-sigs/kueue/blob/main/docs/README.md

File: site/config.toml (hunk 5 above)
[the remark this answers is not in the document]
Comment (alculquicondor) [reply site/config.toml:86]: Can we leave this field empty or remove it?

File: site/content/en/_index.html [hunk 10]
```diff
@@ -0,0 +1,48 @@
++++
+title = "Kueue"
+linkTitle = "Kueue"
+
++++
+
+{{< blocks/cover title="Welcome to Kueue" image_anchor="top" height="full" color="orange" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="https://github.com/kubernetes-sigs/kueue">
+		Learn More <i class="fas fa-arrow-alt-circle-right ml-2"></i>
```
Comment (alculquicondor) [inline site/content/en/_index.html:10]: Let's link to the existing docs

**Review by alculquicondor — COMMENTED** [review]
/approve

**Pull request merged**